source/release_notes/3.0.7-1.0.rst

```diff
@@ -0,0 +1,24 @@
+.. _3.0.7-1.0:
+
+====================================
+Percona Server for MongoDB 3.0.7-1.0
+====================================
+
+Percona is glad to announce the release of Percona Server for MongoDB 3.0.7 on December 11, 2015. Downloads are available `here <https://www.percona.com/downloads/percona-server-for-mongodb>`_ and from the :ref:`Percona Software Repositories <install>`.
+
+Percona Server for MongoDB is a highly scalable, zero-maintenance downtime database supporting the MongoDB v3.0 protocol and drivers. The latest release is based on `MongoDB 3.0.7 <http://docs.mongodb.org/manual/release-notes/3.0/#oct-13-2015>`_, extending MongoDB with `MongoRocks <http://rocksdb.org>`_ and :ref:`PerconaFT <perconaft>` storage engines, as well as several enterprise features. Percona Server for MongoDB requires no changes to MongoDB applications or code.
+
+New Features
+============
+
+* :ref:`toku-backup` is an open-source hot backup utility for |MongoDB| servers running the :ref:`PerconaFT <perconaft>` storage engine (including |Percona Server for MongoDB|).
+
+* New *Proactive First* lock managed engine support ???
+
+Bug Fixes
+=========
+
+* `PSMDB-18 <https://jira.percona.com/browse/PSMDB-18>`_: Changed the deafult data directory (``dbpath``) to :file:`/var/lib/mongo` for RHEL/CentOS.
```

Comment:
not sure if JIRA links are publicly available, so no point in linking them if not